java/gradle.java/nbproject/project.properties

```diff
@@ -24,4 +24,6 @@ javadoc.apichanges=${basedir}/apichanges.xml
 
 test-unit-sys-prop.test.netbeans.dest.dir=${netbeans.dest.dir}
 test-unit-sys-prop.java.awt.headless=true
-test.use.jdk.javac=true
\ No newline at end of file
+test.use.jdk.javac=true
+spec.version.base=1.19.0
```

Comment:
Why are we moving the Specification-Version out of manifest.mf and use spec.version.base?
I'm just curious.

Reply:
https://netbeans.apache.org/wiki/DevFaqImplementationDependency.asciidoc -- it's a way how to autoincrease spec version when the implementation dependency changes. I will move it back when I finalize the `project.dependency` module (based on feedback / findings when implementing Gradle, I'd allow some laziness, still not unsure how).